src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react';
+import { Todo } from '../../types/todo';
+import users from '../../api/users';
+
+type Props = {
+  toAdd: (toAdd: Todo) => void;
```

Comment:
[CODE STYLE] - Avoid using one-letter variable names. Consider renaming `toAdd` to something more descriptive, like `addTodo`.